src/platform/log.c

```diff
@@ -29,3 +42,19 @@ void log_error(const char *msg, const char *param_str, int param_int)
 {
     SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s", build_message(msg, param_str, param_int));
 }
+
+void log_window_init() {
+    int i = 0;
+    for (; i < 40; i++)
+        memset(log_history_buf[i], 0, 256);
```

Comment:
Use the curly brackets {}, even for oneliners. There's no reason here to declare the i variable outside of the for loop here.